diff -Naur ./.exclude_list ../latest/.exclude_list
--- ./.exclude_list	2020-08-07 06:25:43.229286483 +0000
+++ ../latest/.exclude_list	2020-08-07 06:24:56.749485413 +0000
@@ -1,2 +1 @@
 .git
-.gitignore
diff -Naur ./rpgen ../latest/rpgen
--- ./rpgen	2020-08-07 06:25:34.669323144 +0000
+++ ../latest/rpgen	2020-08-07 06:24:56.749485413 +0000
@@ -17,19 +17,9 @@
 	[ -d .rpgen ] && rm -r .rpgen
 	mkdir .rpgen
 	mkdir .rpgen/rp
-	mkdir .rpgen/remote
 	rpgen_rsync .rpgen/latest
 }
 
-rpgen_initrepo() {
-	if [ -n "$(ls -A "$PWD")" ]; then
-		echo "Directory not empty"
-		exit 1
-	fi
-	mkdir rp
-	mkdir latest
-}
-
 rpgen_project_check() {
 	if [ ! -d .rpgen ]; then
 		echo "Not a rpgen project"
@@ -56,7 +46,7 @@
 }
 
 rpgen_checkout_latest() {
-	rsync --quiet --exclude=.rpgen -av .rpgen/latest/ $PWD/
+	rsync --quiet --exclude=.rpgen --delete -av .rpgen/latest/ $PWD/
 }
 
 rpgen_checkout() {
@@ -91,38 +81,11 @@
 	which diff
 }
 
-rpgen_push() {
-	remote_dir="$(cat .rpgen/remote/"$1")"
-	results="$( \
-		diff --brief .rpgen/rp "${remote_dir}/rp" || : | \
-		grep -v '^Only in .rpgen' | \
-		awk '{print $1}' | \
-		sort -u)"
-	if ( echo $results | grep Only ); then
-		echo "Rebase Mode"
-		read -p "Are you sure you want to continue? <y/N> " prompt
-	elif ( echo $results | grep File ); then
-		echo "!!OVERWRITE MODE!!"
-		read -p "Are you sure you want to continue? <y/N> " prompt
-	else
-		echo "Append Mode, auto continue"
-		prompt="y"
-	fi
-	if [ "$prompt" != "y" ]; then
-	  exit 1
-	fi
-	rsync -avzP .rpgen/rp/ "$remote_dir"/rp/
-	rsync -avzP --delete .rpgen/latest/ "$remote_dir"/latest/
-}
-
 case $1 in
 	"init") rpgen_init "${@:2}";;
-	"initrepo") rpgen_initrepo "${@:2}";;
 	"commit") rpgen_commit "${@:2}";;
 	"checkout") rpgen_checkout "${@:2}";;
 	"diff") rpgen_diff "${@:2}";;
 	"diffstat") rpgen_diffstat "${@:2}";;
 	"toolcheck") rpgen_toolcheck "${@:2}";;
-	"pull") rpgen_pull "${@:2}";;
-	"push") rpgen_push "${@:2}";;
 esac
